app/assets/v2/js/animate-number.js

```diff
@@ -0,0 +1,3 @@
+(function(d){var r=function(b){return b.split("").reverse().join("")},m={numberStep:function(b,a){var e=Math.floor(b);d(a.elem).text(e)}},g=function(b){var a=b.elem;a.nodeType&&a.parentNode&&(a=a._animateNumberSetter,a||(a=m.numberStep),a(b.now,b))};d.Tween&&d.Tween.propHooks?d.Tween.propHooks.number={set:g}:d.fx.step.number=g;d.animateNumber={numberStepFactories:{append:function(b){return function(a,e){var f=Math.floor(a);d(e.elem).prop("number",a).text(f+b)}},separator:function(b,a,e){b=b||" ";
```

Comment:
eslint really hates this file 🤣 I think we'll need to mark it as a pass from the linter somehow

Reply:
This is a minified file so no wonder eslint hates it. @thelostone-mc I think just rename it to animate-number.min.js and it will be taken care of.